upstart_msh/cob-start

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+ROBOT=cob4-1
+DISTRO=indigo
+pcs="
+cob4-2-b1
+cob4-2-t1
+cob4-2-t2
+cob4-2-t3
+cob4-2-s1
+cob4-2-h32"
+
+sleep 5
+for i in $pcs; do 
+    Alive=False
+    Crono=0
+    while [[ $Alive=False && $Crono -le 40 ]]
+    do
+        ping -c 1 -w 3 $i
+        if [ $? -ne 0 ] ; then
+          Crono=$((Crono+1))
+          Timeout=True
+          Alive=False
+          echo $i not found
+        else
+          Alive=True
+          Timeout=False
+          echo $i alive
+          break
+        fi
+        sleep 1
+    done
+    if [ $Crono -ge 40 ] ; then
+      Timeout=True
+      echo $i timeout
+      break
+    else
+      Timeout=False
+    fi  
+done
+function log() {
+  logger -s -p user.$1 ${@:2}
+}
+
+source /etc/ros/$DISTRO/cob.d/setup/setup.sh
+JOB_FOLDER=/etc/ros/$DISTRO/cob.d/launch/robot
+export ROSLAUNCH_SSH_UNKNOWN=1 
+LAUNCH_FILENAME=/tmp/robot.launch.launch
+export ROS_IP=`rosrun robot_upstart getifip eth0`
+export ROS_MASTER_URI=http://$ROS_IP:11311
```

Comment:
does this work too, if other launch files are using hostnames then?

I think since adding the hostnames to /etc/hosts there's no reason any more to use ROS_IP. We should comment this lines